packages/core/src/libraries/hook/utils.ts

```diff
@@ -33,6 +34,7 @@ export const sendWebhookRequest = async ({
       'user-agent': 'Logto (https://logto.io/)',
       ...headers,
       ...conditional(signingKey && { 'logto-signature-sha-256': sign(signingKey, payload) }),
+      'logto-message-id': generateStandardId(),
```

Comment:
when retry to send a request, the message id is expected to be the same as prev request

Reply:
Yes, this is ensured by the `got`.